### ISSUE TITLE
Installer fails when installing dependency

### ISSUE THREAD

**Issue by LightWayUp** [yoshiask/FluentStore#26]
Fluent Store Installer fails with the error:

![Fluent Store Beta installer with an error dialog](https://i.imgur.com/WUERMZ1.png)

> ## Install failed
> 
> HRESULT: 0x80073CF3, Package failed updates, dependency or conflict validation.
> 
> You may run the setup again at another time to complete the installation.

This seems to happen when it is trying to install Microsoft.WindowsAppRuntime when the dependency has already been installed on the system.

![The list of installed packaged apps retrieved with Get-AppxPackage PowerShell cmdlet contains Microsoft.WindowsAppRuntime](https://i.imgur.com/XCUoFtp.png)

**Comment by LightWayUp** [comment]
I just did some more test... Apparently the error occurs for me even when the `Microsoft.WindowsAppRuntime.1.0` dependency packages *weren't* installed prior to running the installer.  
I tested in Windows Sandbox, and the installer works fine, so it's probably an issue with my own corrupted Windows system, not the installer 😅, sorry for the misleading information.

However, I discovered an actual unrelated issue when testing in Windows Sandbox: if Visual C++ Redistributable isn't installed, Fluent Store will crash silently upon launch. Please [see this new issue for the details](https://github.com/yoshiask/FluentStore/issues/27), thanks!

**Comment by Permanently** [comment]
I'm getting the same - did you figure out a workaround, or just reinstall Windows?

**Comment by yoshiask** [comment]
@Permanently Have you tried installing the Visual C++ Redistributable?

**Comment by LightWayUp** [comment]
> I'm getting the same - did you figure out a workaround, or just reinstall Windows?

It was years ago, so I can't quite remember what happened in the end... But I was able to test a new version of Fluent Store (as seen in another issue I opened), so this issue may have been specific to just a version of Fluent Store.

If you're encountering this right now, some things to try are to install a different version, or if that doesn't work, try to install it in a clean virtual machine setup, to see if it's potentially due to your current Windows system. Hope this helps!
